Calling/src/app/App.tsx

```diff
@@ -59,35 +65,53 @@ const App = (): JSX.Element => {
     })();
   }, []);
 
+  const isMobileSession = useIsMobile();
+  const isLandscapeSession = isLandscape();
+  const isAppAlreadyRunningInAnotherTab = useSecondaryInstanceCheck();
+
+  useEffect(() => {
+    if (isMobileSession && isLandscapeSession) {
+      console.log('ACS Calling sample: Mobile landscape view is experimental behavior');
+    }
+  }, [isMobileSession, isLandscapeSession]);
+
+  if (isMobileSession && isAppAlreadyRunningInAnotherTab) {
+    return <PageOpenInAnotherTab />;
+  }
+
   const supportedBrowser = !isOnIphoneAndNotSafari();
   if (!supportedBrowser) {
     return <UnsupportedBrowserPage />;
   }
 
-  if (isMobileSession() || isSmallScreen()) {
-    console.log('ACS Calling sample: This is experimental behaviour');
-  }
-
   switch (page) {
     case 'home': {
-      document.title = `home - ${webAppTitle}`;
+      document.title = `home - ${WEB_APP_TITLE}`;
       // Show a simplified join home screen if joining an existing call
       const joiningExistingCall: boolean = !!getGroupIdFromUrl() || !!getTeamsLinkFromUrl();
       return (
         <HomeScreen
           joiningExistingCall={joiningExistingCall}
           startCallHandler={(callDetails) => {
             setDisplayName(callDetails.displayName);
+
             const isTeamsCall = !!callDetails.teamsLink;
-            const callLocator =
-              callDetails.teamsLink || getTeamsLinkFromUrl() || getGroupIdFromUrl() || createGroupId();
-            setCallLocator(callLocator);
+            const makeLocator = (
+              teamsLink?: TeamsMeetingLinkLocator | undefined,
+
+              outboundParticipants?: string[]
```

Comment:
outboundParticipants is fishy. Is this related to the still in beta PSTN calling feature?